blueocean-dashboard/src/main/js/components/karaoke/components/LogConsole.jsx

```diff
@@ -26,19 +25,20 @@ export class LogConsole extends Component {
     }
 
     componentWillMount() {
-        this._processLines(this.props.logArray);
+        // We need a shallow copy of the ObservableArray to "cast" it down to normal array
+        const lineArray = this.props.logArray !== undefined && !Array.isArray(this.props.logArray) ? this.props.logArray.slice() : this.props.logArray;
+        logger.debug('isArray props', Array.isArray(this.props.logArray), 'isArray after', Array.isArray(lineArray));
```

Comment:
There had been a reason to not always do `slice` ...but do not remember. Will try and see if it starts ringing a bell.